pkg/html/dynamic/document.go

```diff
@@ -527,6 +529,49 @@ func (doc *HTMLDocument) ClickBySelectorAll(selector values.String) (values.Bool
 	return values.False, nil
 }
 
+func (doc *HTMLDocument) TypeInput(selector values.String, value core.Value) (values.Boolean, error) {
+
+	valStr := value.String()
+
+	res, err := eval.Eval(
+		doc.client,
+		fmt.Sprintf(`
+			var el = document.querySelector(%s);
+			if (el == null) {
+				return false;
+			}
+			el.focus();
+			return true;
+		`, eval.ParamString(selector.String())),
+		true,
+		false,
+	)
+
+	if err != nil {
+		return values.False, err
+	}
+
+	if res.Type() == core.BooleanType && res.(values.Boolean) == values.False {
+		return values.False, nil
+	}
+
+	inputClient := input.NewClient(doc.conn)
```

Comment:
Why are you creating a new client ? Document already has one.

Reply:
Didn't even notice it.